pretty-pipes/pp-lib/src/main/cpp/pp/Hdfs.cpp

```diff
@@ -33,6 +33,9 @@
 // Standard
 #include <fstream>
 
+//Boost
+#include <boost/shared_array.hpp>
+
```

Comment:
Does including `shared_array.hpp` instead of `shared_ptr.hpp` give you anything?

Reply:
Nope. That's an auto-complete mistake.